cmd/init.go

```diff
@@ -62,14 +72,29 @@ func (cli *CLI) init(opts Options) int {
 				}
 
 				if err != nil {
+					fmt.Fprint(cli.outStream, builder.String())
 					return fmt.Errorf("Failed to find a plugin; %w", err)
 				}
 
-				fmt.Fprintf(cli.outStream, "Plugin \"%s\" is already installed\n", pluginCfg.Name)
+				builder.WriteString(fmt.Sprintf("Plugin \"%s\" is already installed\n", pluginCfg.Name))
+				continue
 			}
 
 			if opts.Recursive && !found {
-				fmt.Fprint(cli.outStream, "No plugins to install\n")
+				builder.WriteString("No plugins to install\n")
+			}
+
+			if installed || !opts.Recursive {
+				fmt.Fprint(cli.outStream, builder.String())
```

Comment:
This `installed` flag is shared across all working directories, so the init output after the working directory where the plugin is installed will unintentionally be printed even if none are installed.

Reply:
Ah do you want to only show the directories that installed plugins? I set it up to show all when any plugins have been installed, but either is fine with me

Reply:
Yes, I think the current behavior makes it difficult to understand the conditions under which the messages are shown. For example, consider the following structure:

```
├── aaa  # no plugins
├── bbb  # installed plugins
└── ccc  # no plugins
```

The output in `aaa` is cleared by `builder.Reset()` so nothing is shown. The output in `bbb` is shown because the plugins are installed. The output in `ccc` is shown because the `installed` flag is set.

In this case, it is difficult to understand why the output in `aaa` is not shown and the output in `ccc` is shown.